src/components/Home/Shreyansh/Week_4/components/Feature1/Feature1Style.ts

```diff
@@ -0,0 +1,30 @@
+import { styled } from 'styled-components';
+
+export const Feature1Wrapper = styled.div`
+  .funFactContainer {
```

Comment:
you are using different naming convention in one file you are using hyphen naming in other file you are using camelCasing